lib/services/local/health_check_config_test.go

```diff
@@ -211,5 +486,71 @@ func TestHealthCheckConfigParser(t *testing.T) {
 		resource, err := parser.parse(event)
 		require.NoError(t, err)
 		require.Equal(t, "example", resource.GetMetadata().Name)
+		require.Equal(t, types.KindHealthCheckConfig, resource.GetKind())
 	})
+
+}
+
+func cloneVirtualDefaultDB() *healthcheckconfigv1.HealthCheckConfig {
+	// Clone to avoid race conditions in parallel tests.
+	return proto.Clone(services.VirtualDefaultHealthCheckConfigDB()).(*healthcheckconfigv1.HealthCheckConfig)
 }
+
+func cloneVirtualDefaultKube() *healthcheckconfigv1.HealthCheckConfig {
+	// Clone to avoid race conditions in parallel tests.
+	return proto.Clone(services.VirtualDefaultHealthCheckConfigKube()).(*healthcheckconfigv1.HealthCheckConfig)
+}
```

Comment:
The `services.VirtualDefaultHealthCheckConfigFoo()` functions return a fresh copy every time, we don't need a `proto.Clone`.